### PR TITLE
Reduce technical debt - cyclomatic complexity

### PR DESCRIPTION
**Pull request by colinbruce**
### Why?

We agreed that reducing the technical debt was a good thing and that we would aim to remove the rubocop_todo file.  This removes the cyclomatic complexity cop and addressed the issues that we weer ignoring

### CODE REVIEW

File: app/models/claims/sort.rb [hunk 1]
```diff
@@ -13,17 +13,8 @@ def sort(column, direction)
...
-    when 'advocate'
-      sort_advocates(direction)
-    when 'case_type'
-      sort_case_type(direction)
-    when 'total_inc_vat'
-      sort_total_inc_vat(direction)
-    when 'amount_assessed'
-      sort_amount_assessed(direction)
+    if META_SORT_COLUMNS.include?(column)
+      send("sort_#{column}", direction)
```
Comment (jsugarman) [inline app/models/claims/sort.rb:17]: :thumbsup: cool

File: app/models/allocation.rb [hunk 2]
```diff
@@ -68,11 +68,19 @@ def case_worker
     # deallocation will have a nil case worker id
   end
 
-  def allocating?
-    @allocating
+  private
+
+  def should_reallocate_claims
```
Comment (jsugarman) [inline app/models/allocation.rb]: Just a naming suggestion: personally like the boolean `?` thing
```
can_allocate_claims?
```

**Review by jsugarman — APPROVED** [review]
Good stuff

File: app/models/allocation.rb (hunk 2 above)
Comment (colinbruce) [reply app/models/allocation.rb]: Good call... 
> naming things is hard

:(

**Pull request merged**